specifications/api/swagger.yaml

```diff
@@ -2226,6 +2224,9 @@ paths:
         <p align="right">status: <strong>stable</strong></p>
 
         Delegate all (current and future) addresses from the given wallet to the given stake pool.
+
+        <strong>Note:</strong> Stake pool id length can be undeterministic when encoded as bech32.
```

Comment:
can be --> is ?

Reply:
I'm not sure that "undeterministic" (which should probably be "nondeterministic") is the best way to word this.

Readers might infer that the bech32 encoding is nondeterministic in its length. Of course this is not true, as Bech32 encoding is completely deterministic.

I would say something like:

```
Bech32-encoded stake pool identifiers can be of multiple different lengths.
```

And perhaps then give some examples to illustrate the different lengths that are possible.